_data/master/navbars/getting-started.yml

```diff
@@ -15,28 +15,27 @@ toc:
     section:
     - title: Introduction
       path: /getting-started/kubernetes/installation/
-    - title: Hosted (via kubectl)
-      section:
-      - title: Introduction
-        path: /getting-started/kubernetes/installation/hosted/
-      - title: Standard Hosted Install
-        path: /getting-started/kubernetes/installation/hosted/hosted
-      - title: Kubeadm Hosted Install
-        path: /getting-started/kubernetes/installation/hosted/kubeadm/
-      - title: Kubernetes Datastore Hosted Install
-        path: /getting-started/kubernetes/installation/hosted/kubernetes-datastore/
-      - title: Canal/flannel Hosted Install
-        path: /getting-started/kubernetes/installation/hosted/canal/
-    - title: Integration Guide
+    - title: Calico networking manifests
```

Comment:
I think we should call this "Installing Calico" and the flannel section should be "Installing Canal".

Then, maybe the "Other networking manifests" should be titled "Installing Calico (policy-only, advanced)", since we often refer to this as "policy-only mode".

I think the current names might be confusing, and we really want to funnel people towards the "Installing Calico" section, so keeping it a clean and simple as possible is good.